datahub_sap_hana/ingestion.py

```diff
@@ -63,10 +88,15 @@ class HanaConfig(BasicSQLAlchemyConfig):
     """Represents the attributes needed to configure the SAP HANA DB connection"""
 
     scheme = "hana"
-    schema_pattern: AllowDenyPattern = Field(default=AllowDenyPattern(deny=["*SYS*"]))
-    include_view_lineage: bool = Field(
+    schema_pattern: AllowDenyPattern = PydanticField(
+        default=AllowDenyPattern(deny=["*SYS*"])
+    )
+    include_view_lineage: bool = PydanticField(
```

Comment:
There is a merge conflict with main here, can you update your branch? Also make sure to run `poetry lock` because there is also a conflict in the lock file.